### ISSUE TITLE
Data not updating, only values from initial adding are available

### ISSUE THREAD

**Issue by kosch104** [itchannel/fordpass-ha#416]
Since the API change I have the issue, that data is only fetched when initially adding my car after I entered my login details. This happened with all past versions, still with 1.60. I can only force refresh the values by removing and re-adding the integration. (UK & Europe)

Also I can't find any service called "fordpass.refresh_status" or anything similar to manually update, as written in the readme.

**Comment by tobox** [comment]
I have the same problem. I managed to call "fordpass.poll_api" and "fordpass.refresh_status" using developer tools -> services, but no value was updated. 

The SoC ist still an old value and the logbook shows now entries for fordpass. 

EVCC running in parallel gets updates from the API correctly, so my account is not suspended.

Running 1.61 Beta 3

**Comment by itchannel** [comment]
Can you please check your logs for any errors generated.

**Comment by tobox** [comment]
The output is very verbose, but I don't see any real errors.  Since the whole log contains sensitive information, I could send it directly if you are interested.

In the log I found this:
... 'xevBatteryStateOfCharge': {'updateTime': '2023-10-30T01:46:45Z', 'oemCorrelationId': '28553', 'value': 95.5},...

The SoC in the Ford App says 95%. 

But in Home Assistent, I only see sensor.fordpass_elveh with attribute "Battery Charge: 88.5".

Does this help? At which sensor/attribute should the current SoC end up?

**Comment by SquidBytes** [comment]
You can provide your json data, with sensitive information redacted, by following: https://github.com/itchannel/fordpass-ha/wiki/AutonomicData-Tutorial

For any error logs, you will still need to provide any error logs returned from HomeAssistant.

For clarification:
`fordpass_elVeh` "Battery Charge"  attribute is returning the value of xevBatteryStateOfCharge in the json - that attribute should be the High Voltage Battery %. If you are charging, or recently drove, and your HomeAssistant API polling is long - it won't match exactly in the FordPass app because the HA integration may not have updated yet.

Regarding the Data not updating, when you initially setup the integration you should have setup a polling time. That time is what will update the information in the integration. Some sensors and attributes change more frequently.
If you go for a drive, and the odometer is not updating, then we would need your logs to see if there is an error somewhere.

**Comment by CptFry** [comment]
I had this earlier too. After I installed the repo, added the card it did work. I then noticed it wasn't updating. 
After restarting HA it updated and has since been updating as per the service interval set.

**Comment by tobox** [comment]
Even after a whole night and driving to work this morning, all displayed sensor values remained at their old values. 

Polling time was set to 900 seconds.

Then I rebooted home assistant and all values were updated!

Strange.

**Comment by itchannel** [comment]
If you are still experiencing issues after updating to 1.61 can you please follow the below tutorial to provide me with your logs so we can diagnose the issue:
https://github.com/itchannel/fordpass-ha/wiki/Obtaining-Integration-Logs

**Comment by tobox** [comment]
With 1.61 (release) it seems to be working properly.

**Comment by SquidBytes** [comment]
Sounds good. Going to close this issue.
If something happens, feel free to open a new issue.
